docs/sources/user/Using-psort.md

```diff
@@ -1,30 +1,39 @@
 # Using psort.py (Plaso Síar Og Raðar Þessu)
 
-**psort** is a command line tool to post-process plaso storage files. It allows you to filter, sort and run automatic analysis on the contents of plaso storage files.
+**psort** is a command line tool to post-process Plaso storage files. It allows
+you to filter, sort and run automatic analysis on the contents of Plaso storage
+files.
 
 Looking for [tips and tricks](Using-psort.md#how-do-i)?
 
 ## Usage
 
-To see a list of all available parameters you can pass to psort use ``-h`` or ``--help``.
+To see a list of all available parameters you can pass to psort use ``-h`` or
+``--help``.
 
-The simplest way to run the tool is simply provide it with a storage file.
+The most basic way to use **psort** is to provide it with a storage file and
+output file, for example:
 
 ```
 $ psort.py -w test.log test.plaso
 ```
 
-This will use the default output module and print out to STDOUT a list of all extracted events, merging detected duplicate events. All timestamps on the output will be in UTC.
+**Note that as of 1.5.0 psort no longer supports wrting output to stdout.**
```

Comment:
☝️ Typo

Reply:
thx for flagging, note that it helps to point out which word is incorrectly written.